releasetasks/templates/notification/notifications.yml.tmpl

```diff
@@ -1,7 +1,7 @@
 {#
 Notifications via Pulse-Notify service
 The macro call should look like
-    task_notifications(taskname, completed=["releasetasks"], failed=["releasetasks"], exception=["releasetasks"])
+    task_notifications(taskname, failed=["releasetasks"], exception=["releasetasks"])
```

Comment:
I don't think you need this change. We still accept this arg

Reply:
Good catch!